java-frontend/src/test/java/org/sonar/java/se/checks/ConditionAlwaysTrueOrFalseCheckTest.java

```diff
@@ -34,6 +36,14 @@ public void test() {
     JavaCheckVerifier.verify("src/test/files/se/ConditionAlwaysTrueOrFalseCheck.java", new ConditionalUnreachableCodeCheck(), new BooleanGratuitousExpressionsCheck());
   }
 
+  @Test
+  public void test_without_jsr305() {
+    Collection<File> classpath = JavaCheckVerifier.classpathFromDirectory("target/test-jars").stream()
+      .filter(file -> !(file.getName().startsWith("jsr305-") || file.getName().startsWith("annotations-")))
+      .collect(Collectors.toList());
```

Comment:
This test is not very robust : you should only include the class you need to have your test case happening.